app/controllers/projects_controller.rb

```diff
@@ -30,6 +30,39 @@ def submit_payment
     @project = Project.find(params[:project_id])
   end
 
+  def pay
+    @project = Project.find(params[:project_id])
+    if @project.developers.any? {|dev| dev.status == 'unavailable'}
```

Comment:
Space between { and | missing.<br>Space missing inside }.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.